docs/model_configuration.rst

```diff
@@ -555,3 +555,20 @@ or when it first receives inference requests. TensorRT optimizations
 include specializing and fusing model layers, and using reduced
 precision (for example 16-bit floating-point) to provide significant
 throughput and latency improvements.
+
+.. _section-model-warm-up:
+
+Model Warm Up
+-------------------
+
+For some framework backends, model initialization may be delayed until the
+first inference is requested, TF-TRT optimization for example, which introduces
+unexpected latency seen by the client. The model configuration
+:cpp:var:`ModelWarmUp <nvidia::inferenceserver::ModelWarmUp>` is used to specify
+warm up settings for a model. The settings define an sample inference request
```

Comment:
"The settings define the inference request that the inference server should create to warm-up each model instance."